addons/mail/models/discuss/discuss_template.py

```diff
@@ -0,0 +1,47 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import api, fields, models
+
```

Comment:
This model and the corresponding wizard seem very similar to the `mail.template` machinery.
What do you guys think @reth-odoo @tde-banana-odoo ?